app/src/main/java/com/example/playlistmaker/media/fragment/MediaLibraryFragment.kt

```diff
@@ -0,0 +1,51 @@
+package com.example.playlistmaker.media.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.playlistmaker.R
+import com.example.playlistmaker.databinding.FragmentMediaLibraryBinding
+import com.example.playlistmaker.media.ui.MediaLibraryAdapter
+import com.google.android.material.tabs.TabLayoutMediator
+
+class MediaLibraryFragment : Fragment() {
+
+    private var _binding: FragmentMediaLibraryBinding? = null
+    private val binding get() = _binding?: throw IllegalStateException("Error")
```

Comment:
`IllegalStateException("Error")` не является говорящей ошибкой, лучше оставить вот так:
```suggestion
    private val binding get() = _binding!!
```